Makefile

```diff
@@ -139,14 +142,17 @@ clean:
 	rm -f node_modules/.uptodate
 	rm -rf build
 
-DOCKER_TAG = dev
-
-GULP := node_modules/.bin/gulp
-
 build/manifest.json: node_modules/.uptodate
 	$(GULP) build
 
 node_modules/.uptodate: package.json
 	@echo installing javascript dependencies
 	@node_modules/.bin/check-dependencies 2>/dev/null || npm install
 	@touch $@
+
+${PYTHON}:
+	pyenv install ${PYTHON_VERSION}
+	pip install --upgrade pip
+
+${TOX}: ${PYTHON}
+	pip install tox
```

Comment:
And here are the actual recipes for installing Python and tox, tucked away at the bottom of the file along with the other intermediate recipes.